src/components/layout/pages.tsx

```diff
@@ -0,0 +1,85 @@
+// Components
+import Icon from "@components/global/icon";
+
+/**
+ * Navigation items seperated into groups
+ * 
+ * For Navigation Drawers (`NavDrawer` from `nav-drawer.tsx`)
+ */
+export const navGroups = [
```

Comment:
maybe turn this into a generator function to create `navGroup` because some URLs would be dynamic

Reply:
I have no idea what that means :(

Reply:
For example,
```js
export function createNavGroup(userData) {
   return [
        ...,
        {
        name: "Your Class",
        icon: <MaterialIcon icon="groups" />,
        url: `/class/1/${userData.class}`,
      },
     ...
]
}
```

Reply:
Can I just open an issue for now? This seems a bit far in the future.

Reply:
or you could make a function that return a variable for now